packages/data/abstract/userpoll/lib/src/poll.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:meta/meta.dart';
+import 'package:poll_model/poll_model.dart';
+
+part 'poll.g.dart';
+
+@JsonSerializable()
+class UserPoll {
+  PollModel mood;
```

Comment:
We have those values hardcoded, that is fine for now. But in future we might have more that one model of poll question per prophecy (one to many). And this might be refactored. 
This is just for your information.

Reply:
I asked about it in chat. When it will be needed we just add other fields and enums.